### ISSUE TITLE
Error Running "Install the Python SNI python-pip dependencies"

### ISSUE THREAD

**Issue by stnguyen90** [atosatto/ansible-dockerswarm#41]
On Ubuntu 14.04.5 LTS, I got the error below when running task "Install the Python SNI python-pip dependencies."
```
Downloading/unpacking pyopenssl
  Downloading pyOpenSSL-17.5.0-py2.py3-none-any.whl (53kB): 53kB downloaded
Requirement already satisfied (use --upgrade to upgrade): six>=1.5.2 in /usr/lib/python2.7/dist-packages (from pyopenssl)
Downloading/unpacking cryptography>=2.1.4 (from pyopenssl)
  Downloading cryptography-2.1.4.tar.gz (441kB): 441kB downloaded
  Running setup.py (path:/tmp/pip_build_root/cryptography/setup.py) egg_info for package cryptography
    error in cryptography setup command: Invalid environment marker: python_version < '3'
    Complete output from command python setup.py egg_info:
    error in cryptography setup command: Invalid environment marker: python_version < '3'

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_root/cryptography
Storing debug log for failure in /home/vagrant/.pip/pip.log
```

**Comment by atosatto** [comment]
Hi @stnguyen90!
What version of Ansible are you running? What version of Python is running on your host?
Can you share the content of the `/home/vagrant/.pip/pip.log` file?

**Comment by stnguyen90** [comment]
I don't have it at the moment, but I think upgrading setuptools should fix the problem.

**Comment by atosatto** [comment]
I this this is fixed by `v2.0.0`. In case this issue is still relevant in your setup, feel free to reopen this issue.
